jujugui/static/gui/src/app/components/store/store.js

```diff
@@ -22,34 +22,818 @@ YUI.add('store', function() {
+                      alt="" />
+                </div>
+                <div className="five-col no-margin-bottom last-col">
+                    <p>Bundles are collections of charms that link
+                    applications together, so you can deploy whole
+                    chunks of infrastructure in one go.</p>
+                    <a href="/?search=&type=bundle"
+                      className="button--inline-neutral">
+                      View all the bundles
+                    </a>
+                </div>
+            </div>
+            {this._tagsSection()}
+        </div>
+      </div>);
+    },
+
+    /**
+      The content for the operations section of the page
+
+      @method _operationsSection
+      @return {Object} The contents of the section
+    */
+    _operationsSection: function() {
+      return (<div className="row">
+        <div className="inner-wrapper">
+            <h2>Operations</h2>
+            <div className="box box--nagios clearfix">
+                <div className="six-col no-margin-bottom align-center">
+                    <img
+                      src={this._generateLocalImagePath('nagios-promo.png')}
+                      alt="" />
+                </div>
+                <div className="six-col no-margin-bottom last-col">
+                    <h3>Nagios</h3>
+                    <p>
+                      By <a href={this._generateUserPath('charmers')}>
+                        charmers
+                      </a>
+                    </p>
+                    <p>Nagios offers complete monitoring, management
+                    and alerting of any service from the charm store
+                    that is related to it.</p>
+                    <a href="/?store=nagios"
+                      className="button--inline-neutral">
+                      View the charm
+                    </a>
+                </div>
+            </div>
+
+            <ul className="twelve-col no-bullets equal-height">
+                <li className="three-col featured-entity">
+                    <a className="featured-entity__link" href="/?store=kibana">
+                        <div className="one-col no-margin-bottom">
+                            <img src={this._generateIconPath('kibana')}
+                              alt=""
+                              className="featured-entity__image" />
+                        </div>
+                        <div
+                          className="two-col last-col no-margin-bottom">
+                            <h3 className="featured-entity__title">
+                              kibana
+                            </h3>
+                            <p>by containers</p>
+                        </div>
+                    </a>
+                </li>
+                <li className="three-col featured-entity">
+                    <a className="featured-entity__link"
+                      href="/?store=logstash">
```

Comment:
That is due to the staging charmstore being out of with live. http://demo.jujugui.org/?store=logstash